content/pages/healthcare/health-conditions/mental-health.md

```diff
@@ -29,13 +29,24 @@ If you qualify for VA health care, you can get high-quality mental health servic
 
 </div>
 
-**Are you in crisis? You have options.** [UX/DESIGN NOTE: WE'D LIKE THIS CRISIS CALLOUT TO BE AN ACCORDION PLEASE.]
+<div class="usa-alert usa-alert-warning va-alert">
+  <div class="usa-alert-body">
+	<h5>Are you in crisis? <a id="crisis-expander-link">You have options.</a></h5>
+	<div id="apply-expander-content" class="expander-content expander-content-closed">
```

Comment:
@cvalarida I was holding onto `apply-expander-content` only because it was tied to the animation. BUT, it looks like that style just got rolled into the `.expander-content` class [here](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/sass/_shame.scss#L166), so can do!